Description: Upstream changes introduced in version 1.1.3-1squeeze5
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 vlc (1.1.3-1squeeze5) stable-security; urgency=high
 .
   * Non-maintainer upload by the Security Team.
   * Fix heap-based buffer overflow in mp4 decoder, VideoLAN-SA-1103.
 .
 The person named in the Author field signed this changelog entry.
Author: Nico Golde <nion@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- vlc-1.1.3.orig/modules/demux/mp4/libmp4.c
+++ vlc-1.1.3/modules/demux/mp4/libmp4.c
@@ -2247,7 +2247,7 @@ static int MP4_ReadBox_frma( stream_t *p
 
 static int MP4_ReadBox_skcr( stream_t *p_stream, MP4_Box_t *p_box )
 {
-    MP4_READBOX_ENTER( MP4_Box_data_frma_t );
+    MP4_READBOX_ENTER( MP4_Box_data_skcr_t );
 
     MP4_GET4BYTES( p_box->data.p_skcr->i_init );
     MP4_GET4BYTES( p_box->data.p_skcr->i_encr );
